.github/workflows/lua-tz.yml

```diff
@@ -0,0 +1,135 @@
+name: lua-tz
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - dependencies/lua-tz/**
+  push:
+    branches:
+      - develop
+      - dev-[2-9][0-9].[0-9][0-9].x
+      - master
+      - "[2-9][0-9].[0-9][0-9].x"
+    paths:
+      - dependencies/lua-tz/**
+
+jobs:
+  get-version:
+    uses: ./.github/workflows/get-version.yml
+
+  package:
+    needs: [get-version]
+
+    strategy:
+      fail-fast: false
+      matrix:
+        distrib: [el8, el9, bullseye, bookworm, jammy]
+        include:
+          - package_extension: rpm
+            image: packaging-stream-connectors-nfpm-alma8
+            distrib: el8
+          - package_extension: rpm
+            image: packaging-stream-connectors-nfpm-alma9
+            distrib: el9
+          - package_extension: deb
+            image: packaging-stream-connectors-nfpm-bullseye
+            distrib: bullseye
+          - package_extension: deb
+            image: packaging-stream-connectors-nfpm-bookworm
+            distrib: bookworm
+          - package_extension: deb
+            image: packaging-stream-connectors-nfpm-jammy
+            distrib: jammy
+
+    runs-on: ubuntu-22.04
+
+    container:
+      image: ${{ vars.DOCKER_INTERNAL_REGISTRY_URL }}/${{ matrix.image }}:${{ needs.get-version.outputs.version }}
+      credentials:
+        username: ${{ secrets.DOCKER_REGISTRY_ID }}
+        password: ${{ secrets.DOCKER_REGISTRY_PASSWD }}
+
+    name: package ${{ matrix.distrib }}
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Checkout luatz sources
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          repository: "daurnimator/luatz"
+          path: "luatz-src"
+          ref: "v0.4-1"
```

Comment:
is seems possible to enforce commit hash instead 🤔 
e.g 0v.4-1 = e49b496e112ae1f0efdec24fc1c6a6f978f68014
![image](https://github.com/centreon/centreon-collect/assets/34628915/2465da18-425d-4ef0-ae09-fd1489bd6b86)

Reply:
I am not sure the benefit of adding a commit hash here ?